testing/libguestfs/APKBUILD

```diff
@@ -0,0 +1,114 @@
+# Contributor:
+# Maintainer: Fathi Boudra <fathi.boudra@linaro.org>
+pkgname=libguestfs
+pkgver=1.40.2
+pkgrel=0
+pkgdesc="libguestfs"
+url="http://libguestfs.org/"
+arch="x86_64"
+license="GPL2"
+depends="cdrkit curl gptfdisk qemu-img qemu-system-x86_64"
+makedepends="
+ acl-dev
+ augeas-dev
+ bash
+ bison
+ cdrkit
+ coreutils
+ cpio
+ file-dev
+ flex
+ fuse-dev
+ gawk
+ gettext-dev
+ hivex-dev
+ jansson-dev
+ gperf
+ gptfdisk
+ libcap-dev
+ libconfig-dev
+ libselinux-dev
+ libselinux-utils
+ libtirpc-dev
+ libvirt-dev
+ libxml2-dev
+ libxml2-utils
+ ncurses-dev
+ ocaml
+ ocaml-findlib
+ pcre-dev
+ perl
+ psmisc
+ python3-dev
+ qemu-img
+ qemu-system-x86_64
+ readline-dev
+ xz
+ xz-dev
+"
+install=""
+subpackages="$pkgname-dev $pkgname-doc $pkgname-lang py3-$pkgname:_py"
+source="
+ http://download.libguestfs.org/1.40-stable/$pkgname-$pkgver.tar.gz
+ https://raw.githubusercontent.com/alpinelinux/aports/3.9-stable/main/kbd/error.h
+"
+builddir="$srcdir/$pkgname-$pkgver"
+options="charset.alias"
+
+prepare() {
+	cd "$builddir"
+	# busybox gzip does not support --best
+	sed -i 's/gzip --best/gzip -9/g' test-data/files/*
+	# musl lacks error.h
+	cp -a "$srcdir"/error.h "$builddir"/lib/
+}
+
+build() {
+	cd "$builddir"
+	PYTHON=/usr/bin/python3 ./configure \
+		--prefix=/usr \
+		--mandir=/usr/share/man \
+		--sysconfdir=/etc \
+		--disable-probes \
+		--disable-daemon \
+		--disable-appliance \
+		--disable-ocaml \
+		--disable-perl \
+		--disable-ruby \
+		--disable-haskell \
+		--disable-php \
+		--disable-erlang \
+		--disable-lua \
+		--disable-golang \
+		--disable-gobject \
+		--without-java
+	# Building index-parse.c and index-scan.c by hand works around a race
+	# condition in the autotools cruft, where two other more copies of yacc
+	# race with each other, resulting in a corrupted file.
+	make -j1 -C builder index-parse.c
+	make -j1 -C builder index-scan.c
+	make -j$(nproc) V=1 INSTALLDIRS=vendor LTLIBINTL=-lintl
```

Comment:
abuild will use the amount of processes specified in /etc/abuild.conf, you should not specify it here (imagine someone not wanting to use all CPUs when building).